apps/namehashlabs.org/components/3 - organisms/role-page.tsx

```diff
@@ -54,15 +52,13 @@ const JoinOurTeam = () => {
           </Link>
         </p>
       </div>
-      <a
+      <Link
         href="mailto:hello@namehashlabs.org"
         className="px-4 cursor-pointer flex items-center gap-3 py-2 rounded-[8px] bg-white flex-nowrap hover:bg-gray-100 transition-colors duration-200 w-full justify-center "
```

Comment:
I think we shouldn't need most / all of these classnames anymore now that this transitioned from an `<a` to one of our new `<Link`?

It's a little difficult to test this right now because we purposefully made the roles data structure empty which means you can't access this page currently.

Reply:
Yeah, so the initiative I took here was to update it to include the correct component, but keep the style implementation, so we can remove the styles if it looked ok. I was going to ask about the role data.

I can add some dummy data to test it out regardless. Will report back!